pkg/app/formatter/json/context.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2022 The envd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package json
+
+import (
+	"fmt"
+
+	"github.com/tensorchord/envd/pkg/types"
+)
+
+type contextJSONInfo struct {
```

Comment:
I think we don't need to add `JSON` to the name since it's already under the json package and it's private.